test/tokenbridge.test.ts

```diff
@@ -108,33 +137,33 @@ describe('TokenBridge', async function() {
 
     l2user = await starknet.deployAccount("OpenZeppelin");
 
-    TokenBridgeL2 = await starknet.getContractFactory('rewaave/token_bridge');
-    tokenBridgeL2 = await TokenBridgeL2.deploy();
+    l2TokenBridgeFactory = await starknet.getContractFactory('rewaave/token_bridge');
+    l2TokenBridgeImpl = await l2TokenBridgeFactory.deploy();
 
-    ProxyFactoryL2 = await starknet.getContractFactory('proxy');
-    proxyTokenBridgeL2 = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
-    proxyL2TokenDai = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
-    proxyL2TokenUsdc = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
+    l2ProxyFactory = await starknet.getContractFactory('proxy');
+    l2TokenBridgeProxy = await l2ProxyFactory.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
+    l2StaticADaiProxy = await l2ProxyFactory.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
+    l2StaticAUsdcProxy = await l2ProxyFactory.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
 
     const rewAaveContractFactory = await starknet.getContractFactory('rewAAVE');
-    rewAaveTokenL2 = await rewAaveContractFactory.deploy({
+    l2rewAAVE = await rewAaveContractFactory.deploy({
       name: 444,
       symbol: 444,
       decimals: 8,
-      initial_supply: {high: 0, low: 1000},
+      initial_supply: {high: 0, low: 0},
       recipient: BigInt(l2user.starknetContract.address),
-      owner: BigInt(proxyTokenBridgeL2.address),
+      owner: BigInt(l2TokenBridgeProxy.address),
     });
 
-    L2TokenFactory = await starknet.getContractFactory('ETHstaticAToken');
-    l2tokenDai = await L2TokenFactory.deploy();
-    l2tokenUsdc = await L2TokenFactory.deploy();
+    l2TokenFactory = await starknet.getContractFactory('ETHstaticAToken');
+    l2StaticADaiImpl = await l2TokenFactory.deploy();
+    l2StaticAUsdcImpl = await l2TokenFactory.deploy();
 
     [signer, l1user, proxyAdmin] = await ethers.getSigners();
 
     pool = await ethers.getContractAt("LendingPool", LENDING_POOL)
     incentives = await ethers.getContractAt("IncentivesControllerMock", INCENTIVES_CONTROLLER)
-    rewAaveTokenL1 = await ethers.getContractAt("ERC20Mock", await incentives.REWARD_TOKEN());
+    l1AAVE = await ethers.getContractAt("ERC20Mock", await incentives.REWARD_TOKEN());
```

Comment:
detail: reward token's symbol is stkAAVE and not AAVE, see https://etherscan.io/address/0x4da27a545c0c5b758a6ba100e3a049001de870f5